docs/cli-tools.md

```diff
@@ -0,0 +1,67 @@
+
+# Command line tools for Openshidft and Kubernetes
+
+This tools allow users to run Kubernetes and Openshift locally, run commands against Kubernetes clusters, and debug workloads running localy or on remote clusters.
+
+## kubectl
+
+The Kubernetes command-line tool, [kubectl](https://kubernetes.io/docs/reference/kubectl/kubectl/), allows you to run commands against Kubernetes clusters. You can use kubectl to deploy applications, inspect and manage cluster resources, and view logs. For more information including a complete list of kubectl operations, see the [kubectl reference documentation](https://kubernetes.io/docs/reference/kubectl/).
+
+``` bash
```

Comment:
low level installation examples may age quickly - perhaps it would be better to link to the (current) official installation guide? (applies also to items below)

Reply:
makes sense to me too,
a. added links to official installation URLs
b. left current bash scripts as TL;DR examples